lib/src/geometry/linear_function.dart

```diff
@@ -0,0 +1,44 @@
+import 'dart:math';
+
+import '../../extensions.dart';
+
+/// This represents a linear function on the ax + by = c form
+class LinearFunction {
```

Comment:
is Line a better name? since everything else is in the geometry lens I think it's more natural.

Reply:
Good suggestion, fixed